src/keys/mod.rs

```diff
@@ -0,0 +1,84 @@
+//! Contains information for keybindings.
+
+use std::collections::HashMap;
+use std::sync::{Arc, RwLock};
+use std::hash::{Hash, Hasher};
+
+use rustwlc::xkb::{Keysym, NameFlags};
+use rustwlc::types::*; // Need * for bitflags...
+
+use super::commands::{self, CommandFn};
+
+mod keypress;
+pub use self::keypress::KeyPress;
+
+mod event;
+pub use self::event::KeyEvent;
+
+lazy_static! {
+    static ref BINDINGS: RwLock<HashMap<KeyPress, KeyEvent>> =
```

Comment:
This isn't used anywhere else, will it be implemented later or is it no longer necessary?